components/contribution-flow/CryptoStepDetails.js

```diff
@@ -0,0 +1,145 @@
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+import { withRouter } from 'next/router';
+import { defineMessages, useIntl } from 'react-intl';
+
+import { TierTypes } from '../../lib/constants/tiers-types';
+
+import { Box } from '../Grid';
+
+import { Label } from '../Text';
+
+import StyledSelect from '../StyledSelect';
+import StyledInputGroup from "../StyledInputGroup";
+
+const messages = defineMessages({
+  cryptoCurrency: {
+    id: `CryptoStepDetails.cryptoCurrency`,
+    defaultMessage: 'Select your currency',
+  },
+  donationAmount: {
+    id: `CryptoStepDetails.donationAmount`,
+    defaultMessage: 'Enter donation amount',
+  },
+});
+
+const options = [
```

Comment:
I think you can drag this to a constants file.

Reply:
Done. 👍🏽